sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/samples/TimeSeriesInsightsClientSample/TimeSeriesInsightsLifecycleSamples.cs

```diff
@@ -26,12 +28,28 @@ public async Task RunSamplesAsync()
             try
             {
                 #region Snippet:TimeSeriesInsightsGetModelSettings
-                
+
                 // Get the model settings for the time series insights environment
-                Response<Models.ModelSettingsResponse> response = await client.GetAsync().ConfigureAwait(false);
-                Console.WriteLine($"Retrieved model {response.Value.ModelSettings.Name}.");
-                
+                Response<TimeSeriesModelSettings> currentSettings = await client.GetModelSettingsAsync().ConfigureAwait(false);
```

Comment:
IIRC we leave out `.ConfigureAwait(false)` in our samples.

Reply:
Resolving as I keep adding changes locally.